core/src/test/scala/com/salesforce/op/ModelInsightsTest.scala

```diff
@@ -474,4 +477,22 @@ class ModelInsightsTest extends FlatSpec with PassengerSparkFixtureTest {
     f0InDer3.variance shouldBe Some(3.3)
   }
 
+  it should "include raw feature distribution information calculated in RawFeatureFilter when it's included" in {
```

Comment:
I think you can omit `when it's included. Perhaps `include raw feature distribution info calculated in RawFeatureFilter`